src/components/Prayers/PrayersList.tsx

```diff
@@ -0,0 +1,14 @@
+import Prayer from 'components/Prayers/Prayer';
+
+export default function PrayersList({ prayers }) {
+  return (
+    <div className="mt-8">
+      {prayers.map((prayer, index) => (
+        <div key={index}>{prayer.blocks[0].data.text}</div>
+      ))}
+
+      {/* TODO: display prayer on a separate page */}
```

Comment:
10/10

Reply:
I'm on it, will be finished on our next coding session